.github/workflows/main.yml

```diff
@@ -42,13 +70,19 @@ jobs:
       with:
         submodules: recursive
 
-    - name: Test workflow
+#    - name: Test workflow
+#      uses: snakemake/snakemake-github-action@v1.23.0
+#      with:
+#        directory: .test
+#        snakefile: workflow/Snakefile
+#        args: "--use-conda --show-failed-logs --cores 2 --conda-cleanup-pkgs cache --all-temp"
```

Comment:
Why is this now commented out??

Reply:
I tried to run both simultaneously, but when the workflow reached 3-prime part, it fails due to lack of data space. So I comment one after it is finished and ran the other.